### PR TITLE
#145819805 : Make TLS configuration (keys, certfiles) optional for the service broker

### PR DESCRIPTION
**Pull request by rohitsharma04**

### CODE REVIEW

File: servicebroker/test/util/apiServerUtil.test.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+'use strict';
```
Comment (kanekoh) [inline servicebroker/test/util/apiServerUtil.test.js]: The file name should use '_' instead of '.' as that is the convention we are following across.

**Comment by ghaih** [comment]
Just a general comment, will be good to have separate stories for the api and service broker. And there is a code refactoring here, so a separate commit for code refactor makes review bit easier. Thanks!

File: api/lib/utils/schedulerUtils.js [hunk 2]
```diff
@@ -5,40 +5,54 @@ module.exports = function(schedulerSettings) {
...
+  var getOptions = function(appId, policy, policyGuid ){
+    var options = { 
+      json: true,
+      timeout: 10000
+    };
+    if(policy && policyGuid){
+      options.body = policy;
+      options.url = schedulerSettings.uri + '/v2/schedules/' + appId + "?guid=" + policyGuid;
+    }else{
+      options.url = schedulerSettings.uri + '/v2/schedules/' + appId;
```
Comment (qibobo) [inline api/lib/utils/schedulerUtils.js]: As policy and policyGuid are required parameter, why "if else " here?

**Comment by rohitsharma04** [comment]
@kanekoh, incorporated changes,
@ghaih, I'll create a new PR for apiserver.

**Comment by boyang9527** [comment]
two things we need to discuss:
1) do we need to check the client is using https to access our server even though the SSL termination has been done by router/lb in the front

2) Do we force to use https when talking to other components

**Pull request merged**